DM-3165: Adding changelogs for DM-3165 migration of springfox to spri… (#2378)
SoftwareAG/c8y-docs: content/change-logs/device-management/content/lwm2m-agent-10-20-377-0-Security-improvement-migration-springfox-to-springdoc.md

---
date:
title: LWM2M service security improvement by replacing deprecated library 
change_type:

>   - value: change-2c7RdTdXo4
>     label: Improvement

Reviewer: ```suggestion
  - value: change-VSkj2iV9m
    label: Fix
```

product_area: Device Management & Connectivity
component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:

>   - value: tc-QHwMfWtBk7
>     label: cumulocity

Reviewer: ```suggestion
  - value: tc-ggH2M4hf3
    label: lwm2m-agent
```

ticket: DM-3165
version: 10.20.377.0
---

> The deprecated Swagger/OpenApi Library used by LWM2M service was replaced by a new supported library to improve security.

Reviewer 2: ```suggestion
To improve security, the deprecated Swagger/OpenAPI library used by the LWM2M service has been replaced by a new supported library.
```

Reviewer 2: From my perspective this is actually a fix. What do you think, @Oezge-Akin ?

Reviewer: yes we can mark it as fix
